src/Select.soy

```diff
@@ -18,18 +19,21 @@
 	<div class="select{$elementClasses ? ' ' + $elementClasses : ''}" data-onkeydown="handleKeyDown_">
 		{let $currSelectedIndex: isNonnull($selectedIndex) ? $selectedIndex : ($label or (length($items) == 0) ? -1 : 0) /}
 		<input
+			disabled="{$disabled}"
 			type="hidden"
 			name="{$hiddenInputName ? $hiddenInputName : ''}"
 			value="{$currSelectedIndex == -1 ? '' : $values ? $values[$currSelectedIndex] : ''}"
 		>
 		{call Dropdown.render}
 			{param body kind="html"}
 				{foreach $item in $items}
-					<li
-						data-onclick="{$handleItemClick_ ?: ''}"
-						data-onkeydown="{$handleItemKeyDown_ ?: ''}"
-						class="select-option{$currSelectedIndex == index($item) ? ' selected' : ''}">
-						<a href="javascript:;">{$item}</a>
+					<li class="select-option{$currSelectedIndex == index($item) ? ' selected' : ''}">
+						<button
```

Comment:
I see... I've forgotten it. I've used button since browsers triggers events to disabled `<a>` tags.